### PR TITLE
Wildfire - add wildfire-get-sample command

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16688

## Description
add wildfire-get-sample command

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - https://github.com/demisto/etc/issues/16700
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Wildfire.yml [hunk 1]
```diff
@@ -1017,7 +1054,48 @@ script:
...
+      type: string
+    - contextPath: File MD5
+      description: MD5 hash of the file
+      type: string
+    - contextPath: File.Extension
+      description: File Extension
+      type: string
+    description: Retrieve a sample from WildFire
   - name: wildfire-upload-file-remote
+    deprecated: true
```
Comment (yuvalbenshalom) [inline Integrations/integration-Wildfire.yml]: should not be deprecated before the new integration is in

Comment (bakatzir) [reply Integrations/integration-Wildfire.yml]: we already have the wildfire-upload-file-url, so should be deprecated I believe.

**Comment by Clairvoyantx** [comment]
Screen Shot 2019-04-24 at 10 37 02 AM
The get-sample command returns an error on not finding a hash.

**Comment by bakatzir** [comment]
@clairvoyantx 
I'll fix that on sunday morning.
Note that non malicious hashes are available for 14 days only.

**Pull request merged**